Sources/TensorFlow/Core/Tensor.swift

```diff
@@ -23,8 +23,12 @@ public protocol AnyTensor {
     var _tensorFlowDataType: TensorDataType { get }
 }
 
-/// `Tensor` is a multi-dimensional array used for computation. It is a wrapper around a
-/// `TensorHandle`.
+/// A multidimensional array of elements that is a generalization of
```

Comment:
The column limit in this repo is 100. Could you please make these comments fit in 100 columns?

Reply:
sure, I have made sure that these comments fit in 100 columns.

Reply:
Although they do fit within 100 columns, I don't believe they fit to size yet. Could you expand each line so that it fits to size?

Reply:
As suggested, each line is expanded so that it fits to size.